docs/documentation/updating-the-kit.md

```diff
@@ -1,129 +1,70 @@
-# Updating the kit
+# Updating to the latest version
 
-**Important note**
+You can update to the latest version of the GOV.UK Prototype Kit if you want to either:
 
-If you have made any changes outside the `app` folder, this process will destroy those changes. We will try and improve the update process to avoid this, but in the meantime you will need to make a note of your changes outside `app`, and add them back after updating.
+- use a new component
+- improve how your prototype meets accessibility requirements
 
-## Steps
+You can update using either:
 
-[Download the latest Prototype Kit](/docs/download) and unzip it.
+- Node.js package manager (npm)
```

Comment:
no people shouldnt do this. If they do they will miss any updates in the kit itself

Reply:
Discussed with Joe and resolved in e87063a